examples/basic_mysql_connection/index.js

```diff
@@ -8,13 +8,22 @@ var connection = mysql.createConnection({
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
-
-connection.query('SELECT * FROM todos', function (error, results, fields) {
+let query=['select count(*) as items from todos',
```

Comment:
We talked about style issues in class - it will absolutely execute the same, but using capital letters for SQL keywords is a convention that dramatically helps with readability. For next time, can you use `SELECT COUNT(*) AS items FROM todos` instead?